tla-io/src/test/scala/at/forsyte/apalache/io/tnt/TestQuint.scala

```diff
@@ -0,0 +1,75 @@
+package at.forsyte.apalache.io.quint
+
+import org.junit.runner.RunWith
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatestplus.junit.JUnitRunner
+
+import at.forsyte.apalache.io.quint.QuintType._
+import at.forsyte.apalache.tla.lir.{FunT1, IntT1, RecRowT1, RowT1, StrT1, TupT1, VarT1, VariantT1}
+
+@RunWith(classOf[JUnitRunner])
+class TestQuint extends AnyFunSuite {
+
+  test("TNT's named type variables are converted into numbered TlaType1 variables") {
```

Comment:
Test descriptions on this file were not updated

Reply:
dang. sorry about the sloppiness :( i'll clean these all up.